packages/terra-button-group/README.md

```diff
@@ -0,0 +1,25 @@
+# Terra Button Group
+
+
+[![NPM version](http://img.shields.io/npm/v/terra-button-group.svg)](https://www.npmjs.org/package/terra-button-group)
+[![Build Status](https://travis-ci.org/cerner/terra-core.svg?branch=master)](https://travis-ci.org/cerner/terra-core)
+
+{insert description}
+
+- [Getting Started](#getting-started)
+- [Documentation](#documentation)
```

Comment:
```
- [Documentation](https://github.com/cerner/terra-core/tree/master/packages/terra-button-group/docs)
```
We want to use the absolute path here so the path works when we publish it to npm.